opendevin/runtime/utils/runtime_build.py

```diff
@@ -231,8 +190,13 @@ def build_runtime_image(
 
     This is only used for **eventstream runtime**.
     """
-    new_image_name = _get_new_image_name(base_image)
-    logger.info(f'New image name: {new_image_name}')
+    new_image_name = get_new_image_name(base_image)
+    if base_image == new_image_name:
+        logger.info(
+            f'Using existing od_runtime image [{base_image}]. Will NOT build a new image.'
+        )
+    else:
+        logger.info(f'New image name: {new_image_name}')
```

Comment:
Just confirming, will this cause any issues? For instance, is there a possibility that the OpenDevin code gets updated, but the image name stays the same so it doesn't build a new image?

Reply:
Seems like a good enough solution for now, happy to merge.

Reply:
Thanks!